packages/framework/request-handler/src/requestHandlers.ts

```diff
@@ -34,7 +34,7 @@ export type RuntimeRequestHandler = (request: RequestParser, runtime: IContainer
  * handling of external URI to internal handle is required (in future, we will support weak handle references,
  * that will allow any GC policy to be implemented by container authors.)
  */
-export const deprecated_innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
+export const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
```

Comment:
This still seems like it'll have the exact same problem of going directly to the handle resolution code instead of our container handles? Is there a different entrypoint we should be using after this?

Reply:
Correct me if I got it wrong, but I think you need reentrancy with request() handling, and you can do it directly against IContainerRuntimeBase. I.e. innerRequestHandler() is one of the handlers that knows how to resolve internal URIs, but any handler can back into IContainerRuntimeBase if it needs to re-enter resolution.

Reply:
So maybe the issue was that we shouldn't have been calling deprecated_innerRequestHandler in that part of the code? If that's the case, we might not need this fix, we might just need to swap the code back to containerRuntime.request() in our code.

Reply:
We have the full containerruntime available to us at the time of calling this. Although I suppose in other parts of our code we do not, so we do need it back on the base if it was unavailable I suppose.

Reply:
I've put request() back on IContainerRuntimeBase, so it's available to anybody. Yes, reentrancy into request() should do it I believe